web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -405,14 +405,9 @@ def test_set_env_diff_gerrit_output(self):
 
         env["CC_CHANGED_FILES"] = changed_file_path
 
-        with self.assertRaises(subprocess.CalledProcessError):
-            out = get_diff_results([self._run_names[0]], [self._local_reports],
-                                   '--unresolved', 'gerrit',
-                                   ["--url", self._url, "-e", export_dir])
-
-            self.assertIn("'CC_REPO_DIR'", out)
-            self.assertIn("'CC_CHANGED_FILES'", out)
-            self.assertIn("needs to be set", out)
+        get_diff_results([self._run_names[0]], [self._local_reports],
```

Comment:
Why it is changed. It should raise an exception if the environment variables are not set. See: 
https://github.com/Ericsson/codechecker/blob/e02baa734ab58c696170c7289477b2ebee1d5b06/web/client/codechecker_client/cmd_line_client.py#L817-L819